test/builtin_tool_stubs/hadolint

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env -S mise tool-stub
+
+version = "2.12.1-beta"
```

Comment:
Had to use this version to avoid a segfault: https://github.com/hadolint/hadolint/issues/897